plugins/catalog-backend/knexfile.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = {
+    development: {
+      client: 'sqlite3',
+      connection: {
+        filename: './dev.sqlite3',
+      },
+      seeds: {
+        directory: './src/database/seeds/',
+        },
+        migrations: {
+            directory: './src/database/migrations/',
+        }
+    }
+  };
+  
```

Comment:
you have to set your editor to reformat on save (indent is a bit off), and also to insert newlines at the end of files. Add the Prettier plugin if you are on vscode